src/app/games/trexo/TrexoState.ts

```diff
@@ -0,0 +1,126 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { GameStateWithTable } from 'src/app/jscaip/GameStateWithTable';
+import { Player, PlayerOrNone } from 'src/app/jscaip/Player';
+import { ArrayUtils } from 'src/app/utils/ArrayUtils';
+import { assert } from 'src/app/utils/assert';
+import { MGPFallible } from 'src/app/utils/MGPFallible';
+
+/**
+ * Represent half a tile
+ * owner: to make us know which half of a tile it is (since trexo tiles are bicolor)
+ * height: allow us to know where on the Z axis they are ... sound kind of bad no ?
+ *     a stack contains [a, b, c], we know a, b and c height no??
```

Comment:
oui la height est pas utile, faut la calculer en dessinant le plateau mais pas l'encoder dans le plateau je dirais

Reply:
it swims with the fishes !

Reply:
good, maintenant faut enlever le commentaire !